platform/android/java_godot_lib_jni.cpp

```diff
@@ -519,7 +519,7 @@ class JNISingleton : public Object {
 				ret = sarr;
 				env->DeleteLocalRef(arr);
 			} break;
-			case Variant::PACKED_REAL_ARRAY: {
+			case Variant::PACKED_FLOAT32_ARRAY: {
```

Comment:
Missing 64-bit.

Reply:
I will add a warning because I have no idea how to do this